.travis.yml

```diff
@@ -2,11 +2,9 @@ language: php
 php:
   - 7.0
   - 5.6
-  - 5.5
```

Comment:
👍 for cleaning up these old versions, could we add more 7.x releases too?

Reply:
7.1 + 7.2 added.